mrbeam_ledstrips/server.py

```diff
@@ -172,10 +173,9 @@ def _socket_monitor(self, server_address, callback):
 						connection.sendall(str("ERROR : error while processing message from client") + '\x00')
 					except:
 						pass
-		except (KeyboardInterrupt, SystemExit):
-			pass
-		except Exception:
-			self.logger.exception("Exception in socket monitor: ")
+		except Exception as e:
```

Comment:
so now we are ok with raising KeyboardInterrupt, SystemExit ?

Reply:
I didn't mention it in the commits - Given that I could read or imagine a reason for why we want to catch these, I preferred to remove them because they only hinder us in trying to kill the LED server when we would like.